src/include/platform/internal/BLEManager.h

```diff
@@ -65,7 +65,9 @@ class BLEManager
     CHIP_ERROR SetDeviceName(const char * deviceName);
     uint16_t NumConnections();
     void OnPlatformEvent(const ChipDeviceEvent * event);
+#if CONFIG_NETWORK_LAYER_BLE
```

Comment:
The whole file is for BLE so excluding just a single method from it seems weird.